MdeModulePkg/Library/BrotliCustomDecompressLib/BrotliDecompress.c

```diff
@@ -182,7 +182,7 @@ BrGetDecodedSizeOfBuf (
   )
 {
   UINT64  DecodedSize;
-  INTN    Index;
+  INT16   Index;

```

Comment:
I think this can be cleaned given `StartOffset` is unsigned up by adjusting the loop logic.

```c
 for (Index = EndOffset; Index > StartOffset; Index--) {
    DecodedSize = LShiftU64 (DecodedSize, 8) + EncodedData[Index - 1];
  }
```

Reply:
Good suggestion. Updated code with suggestion.